1.2-Introduction-to-Musig.ipynb

```diff
@@ -141,9 +141,9 @@
     "\n",
     "In this exercise, we'll generate nonces for individual participants, calculate the nonce point commitments, and then generate an aggregate nonce point.\n",
     "\n",
-    "* Use `generate_schnorr_nonce()` to generate a random nonce. The function returns an `ECkey` object which is a valid schnorr nonce (quadratic residue y-value).\n",
-    "* Use `aggregate_schnorr_nonces()` to aggregate those individual nonces. The function takes a list of `ECKey` nonces and returns a the aggregate nonce and a negation flag:\n",
-    "  * the aggregate nonce is an `ECPubKey` object\n",
+    "* Use `generate_schnorr_nonce()` to generate a random nonce. The function returns an `ECkey` object which is a valid schnorr nonce.\n",
+    "* Use `aggregate_schnorr_nonces(nonce_point_list)` to aggregate those individual nonces. The function takes a list of `ECKey` nonces and returns a the aggregate nonce and a negation flag:\n",
+    "  * the aggregate nonce is a `ECPubKey` object\n",
```

Comment:
nit: this should be 'an `ECPubKey`', not 'a `ECPubKey`'

Reply:
really? ha.

Reply:
Done